src/SFA.DAS.EmploymentCheck.Functions.UnitTests/AzureFunctions/Triggers/ProcessEmploymentChecksHttpTriggerTests/WhenTriggeringHttpProcessEmploymentChecks.cs

```diff
@@ -15,13 +15,12 @@ namespace SFA.DAS.EmploymentCheck.Functions.UnitTests.AzureFunctions.Triggers.Pr
 {
     public class WhenTriggeringHttpProcessEmploymentChecks
     {
-        private Mock<HttpRequestMessage> _request;
-        private Mock<IDurableOrchestrationClient> _starter;
-        private Mock<ILogger> _logger;
-        private Fixture _fixture;
+        private readonly Mock<HttpRequestMessage> _request;
+        private readonly Mock<IDurableOrchestrationClient> _starter;
+        private readonly Mock<ILogger> _logger;
+        private readonly Fixture _fixture;
 
-        [SetUp]
-        public void SetUp()
+        public WhenTriggeringHttpProcessEmploymentChecks()
```

Comment:
This got reverted to a constructor in the master merge, should be a setup method instead

Reply:
Fixed.